### DISCUSSION TITLE
文字显示html标签

### DISCUSSION THREAD

**Discussion by linjinglan** [baidu/amis#3728] [General]
![image](https://user-images.githubusercontent.com/10203241/157364542-f16b1830-0578-4d56-ae3a-d1928cc4d4d9.png)
第一行是直接在文字里写html标签，能正常识别html标签
![image](https://user-images.githubusercontent.com/10203241/157364280-30c3ab0e-74e5-4b53-b4bb-79fde029fe2e.png)

第二行是通过变量绑定，无法识别html标签，直接把html标签也显示出来了
![image](https://user-images.githubusercontent.com/10203241/157364325-b95a6f00-1331-4c88-ae0d-297c4fb6dffb.png)

**Comment by maccy1** [comment]
使用${content|raw}就可以正常显示html标签了
